Build/Props/CodeJam.Default.props

```diff
@@ -10,9 +10,7 @@
 
 		<LangVersion>9.0</LangVersion>
 		<Nullable>enable</Nullable>
-		<AnalysisLevel>latest</AnalysisLevel>
-		<EnableNETAnalyzers>true</EnableNETAnalyzers>
-		<AnalysisMode>AllEnabledByDefault</AnalysisMode>
+		<RunAnalyzers>False</RunAnalyzers>
```

Comment:
Why?

Reply:
1) The analyzers slow down the build significantly.
2) The analyzers produces many false positives.

Reply:
Can you give examples of false positives ?
If these are in tests, it can be suppressed in editorconfig for all tests.
I didn’t see huge slowdown on my machine.
IMO benefits ouweights.
Maybe then we should have build without analyzers and one build only for latest  .NET with ?

Reply:
The Index.ToString method complains that uint.ToString does not specify a locale. And then there is still a slowdown when building.
![image](https://user-images.githubusercontent.com/803486/123264705-f612a500-d502-11eb-9dd6-aaaa473fcc81.png)

On my machine:
No analyzers: Build completed in 00:00:39.355
With analyzers: Build completed in 00:02:50.387

Are there any analyzers that are useful and yet absent from R#?

Reply:
Analyzer gives correct warning. Calling uint.ToString without locale gives unexpected result.